spec/features/system_admin/users/add_a_lead_school_to_user_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "Creating a new lead school for a user" do
```

Comment:
Sorry... really nit picky we've used all lower case for describe|context|feature|scenario|it strings in all of the other specs.

Reply:
Fair enough, I've made the changes =)